### PR TITLE
Enable Sqlite tests, uncommented snapshot tests in Persistence.Testkit

### PR DESCRIPTION
**Pull request by alexvaluyskiy**
Partially fixes https://github.com/akkadotnet/akka.net/issues/2659

### CODE REVIEW

**Comment by Silv3rcircl3** [comment]
@alexvaluyskiy snapshot specs are failing on mono

File: src/core/Akka.Persistence.TestKit.Tests/LocalSnapshotStoreSpec.cs [hunk 1]
```diff
@@ -14,7 +14,8 @@
 
 namespace Akka.Persistence.TestKit.Tests
 {
-    public class LocalSnapshotStoreSpec : SnapshotStoreSpec
+    // FIXME
+    public abstract class LocalSnapshotStoreSpec : SnapshotStoreSpec
```
Comment (Silv3rcircl3) [inline src/core/Akka.Persistence.TestKit.Tests/LocalSnapshotStoreSpec.cs]: What's the reason to make it abstract?

Comment (alexvaluyskiy) [inline src/core/Akka.Persistence.TestKit.Tests/LocalSnapshotStoreSpec.cs]: To skip only LocalSnapshot spec, not all specs inherited from `SnapshotStoreSpec`

**Comment by Silv3rcircl3** [comment]
@alexvaluyskiy now all sqlite specs failed on windows

**Comment by Danthar** [comment]
@alexvaluyskiy @Silv3rcircl3 that might be because the `LocalSnapshotStoreSpec ` was made abstract. XUnit does not like abstract test-specifications.

File: src/contrib/persistence/Akka.Persistence.Sqlite.Tests/Akka.Persistence.Sqlite.Tests.csproj [hunk 2]
```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <AssemblyTitle>Akka.Persistence.Sqlite.Tests</AssemblyTitle>
     <TargetFrameworks>net452;netcoreapp1.1</TargetFrameworks>
-    <RuntimeIdentifier>win7-x64</RuntimeIdentifier>
+    <RuntimeIdentifier Condition=" '$(TargetFramework)' == 'net452' And '$(OS)' == 'Windows_NT'">win7-x64</RuntimeIdentifier>
```
Comment (alexvaluyskiy) [inline src/contrib/persistence/Akka.Persistence.Sqlite.Tests/Akka.Persistence.Sqlite.Tests.csproj]: Workaround

Comment (Aaronontheweb) [inline src/contrib/persistence/Akka.Persistence.Sqlite.Tests/Akka.Persistence.Sqlite.Tests.csproj]: Looks good to me in the short-run - I @heynickc was mentioning this morning that the runtime might be part of the reason why we're having some trouble with the tests...

**Pull request merged**